core/services/gateway/handlers/capabilities/webapi.go

```diff
@@ -1,11 +1,12 @@
 package capabilities
 
 type Request struct {
-	URL       string            `json:"url"`                 // URL to query, only http and https protocols are supported.
-	Method    string            `json:"method,omitempty"`    // HTTP verb, defaults to GET.
-	Headers   map[string]string `json:"headers,omitempty"`   // HTTP headers, defaults to empty.
-	Body      []byte            `json:"body,omitempty"`      // HTTP request body
-	TimeoutMs uint32            `json:"timeoutMs,omitempty"` // Timeout in milliseconds
+	URL              string            `json:"url"`                 // URL to query, only http and https protocols are supported.
+	Method           string            `json:"method,omitempty"`    // HTTP verb, defaults to GET.
+	Headers          map[string]string `json:"headers,omitempty"`   // HTTP headers, defaults to empty.
+	Body             []byte            `json:"body,omitempty"`      // HTTP request body
+	TimeoutMs        uint32            `json:"timeoutMs,omitempty"` // Timeout in milliseconds
+	MaxResponseBytes uint32            `json:"maxBytes,omitempty"`  // Maximum number of bytes to read from the response body
```

Comment:
Could you add a comment saying this is only advisory and that the gateway will apply a lower maximum if that's what it's configured with?